docs/development/quickstart.md

```diff
@@ -302,9 +261,11 @@ that will provide some mocked data.
 1. Make sure you have the latest change for the python testbed server
 
    ```shell
-   POETRY_LIBPARSEC_BUILD_PROFILE=ci poetry install --extra 'backend'
+   POETRY_LIBPARSEC_BUILD_PROFILE=ci poetry install
```

Comment:
```suggestion
./make.py i
```

Not sure why the `ci` profile is specified instead of using the `dev` mode

Reply:
The CI profile provide a slightly optimized lib

Reply:
This is not worth the added complexity, having `./make.py i` as the single command to always use is very convenient for newcomers and people not versed into Rust

We should add complexity that only once we've reach a actual performance issue ^^

Reply:
Note: this dicussion is an outdated change, did you review the latest change ?

Reply:
![image](https://github.com/Scille/parsec-cloud/assets/3187637/f0767f78-a8c7-49fb-bc2f-36bb8c043bce)

I think I did, yes

Reply:
So this is still a single command

Reply:
- we don't need (until further proof is given) the additional perf
- we don't need the additional cognitive load of having to deal with `./make.py python-ci-install` vs `./make.py i`
- we don't need the additional compile time and disk usage when dev switches between `./make.py python-ci-install` vs `./make.py i` (because the dev is not really sure what is the difference, and might be given one or the other command to run depending on who is helping him)
- so just KISS please :pray: 

btw can you be more precise when replying (what have changed/what is the rational/what should we do, instead of short and broad reply that let the reader puzzled).
For instance
> Note: this dicussion is an outdated change, did you review the latest change ?

can be rephrased as 
> I've updated the code there, `./make.py python-ci-install` is still used but I've added a comment to explain the rational. I believe this is worth the added complexity over simply using `./make.py i` because [...]

Reply:
I should have close that discussion, that way you would have reopen a new one and I would have understand the clear intent of the previous comment (<https://github.com/Scille/parsec-cloud/pull/4731#discussion_r1241940786>)

Reply:
I'm now using `make.py i`